src/main/java/appeng/core/AppEng.java

```diff
@@ -92,8 +91,8 @@
 	// + net.minecraftforge.common.ForgeVersion.revisionVersion + '.' // revisionVersion
 	// + net.minecraftforge.common.ForgeVersion.buildVersion + ",)"; // buildVersion
 
-	@Nonnull
-	private static final AppEng INSTANCE = new AppEng();
+	@Instance( MOD_ID )
+	public static AppEng instance;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static instance" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1444)